src/states/Postroll.js

```diff
@@ -0,0 +1,139 @@
+import videojs from 'video.js';
+
+import {AdState, BeforePreroll, Preroll, AdsDone} from '../states.js';
+import {startAdBreak, endAdBreak} from '../adBreak.js';
+
+export default class Postroll extends AdState {
+
+  init(player) {
+    // Legacy name that now simply means "handling postrolls".
+    player.ads._contentEnding = true;
+
+    // Start postroll process.
+    if (!player.ads.nopostroll_) {
+      player.addClass('vjs-ad-loading');
+
+      this._postrollTimeout = player.setTimeout(function() {
+        player.trigger('adtimeout');
+      }, player.ads.settings.postrollTimeout);
+
+    // No postroll, ads are done
+    } else {
+      player.setTimeout(() => {
```

Comment:
Seems to me that this could be a state transition and then just trigger the `ended` event. Thoughts?

Reply:
To summarize previous comment: not sure what the implications are, want to push to possible future work rather than retest.